packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -43,25 +43,47 @@ function composeObjectExpressionPropertyIntoString(
 
 export function getLoadArgument(
   node: TSESTree.MemberExpression
-): string | undefined {
+): string[] | undefined {
   node = findTopLevelExpression(node);
 
   if (
     isLoadFunction(node) &&
     node.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression
   ) {
-    if (node.parent.arguments.length === 0) {
-      return undefined;
-    }
-    if (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal) {
-      return node.parent.arguments[0].value as string;
-    } else if (
-      node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.ObjectExpression
+    if (
+      node.parent.arguments.length === 0 ||
+      (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal &&
+        (node.parent.arguments[0].value as string) === "")
     ) {
-      return composeObjectExpressionPropertyIntoString(
-        node.parent.arguments[0]
-      );
+      return undefined;
     }
+
+    let properties: string[] = [];
+    node.parent.arguments.forEach(
+      (propertiesSeparatedByComma: TSESTree.CallExpressionArgument) => {
```

Comment:
'propertiesSeparatedByComma' is not accurate give it may not be a literal and if it is it may not be separated by commas.  I would use a simple 'argument' given that is what you are examining.